### PR TITLE
Let the Game model changeset determine the winner

### LINKED ISSUE #10: The Matches controller should be able to determine the winner
I shouldn't have to enter the winner when creating a match

### PR DESCRIPTION
**Pull request by Serneum**
Closes #10

### CODE REVIEW

File: web/models/game.ex [hunk 1]
```diff
@@ -26,5 +26,20 @@ defmodule CoverMyPingPong.Game do
...
+  end
+
+  defp put_winner(changeset) do
+    case changeset do
+      %Ecto.Changeset{valid?: true, changes: %{player_id: p_id, player_score: p_score, opponent_id: o_id, opponent_score: o_score}} ->
+        winner = [%{id: p_id, score: p_score}, %{id: o_id, score: o_score}]
+        |> Enum.sort(&(&1.score > &2.score))
+        |> List.first
+
+        IO.inspect winner
```
Comment (jlholm) [inline web/models/game.ex]: Can we ditch this?

Comment (Serneum) [reply web/models/game.ex]: Yeah, thought I did

**Comment by Serneum** [comment]
@JLHOLM this is ready for review/merge

**Review by jlholm — APPROVED** [review]
LGTM 🍾

**Pull request merged**